homework-submission/react-hyf/react-hyf-client/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "react-hyf-client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^4.3.1",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "3.0.1"
```

Comment:
When you install a package make sure you have it in your `package.json` file, when I ran your application first error I found:

<img width="421" alt="Screen Shot 2019-07-17 at 06 46 42" src="https://user-images.githubusercontent.com/25272822/61348044-864c0800-a85f-11e9-9b29-3bd76d306c17.png">

Reply:
By me it's still working 🤔, might i've removed it accidentally from the package.json after i totally finished.